test/mobile/features/fixtures/nativeplugin/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Tue Oct 02 16:02:59 PDT 2018
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```

Comment:
This is quite an old version of gradle - it'd be worth upgrading to 6 unless there's a specific reason 4 is needed, as it should make the build faster.

Reply:
Moved to 6.8.3.